inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java

```diff
@@ -45,24 +45,24 @@ public interface InlongStreamApi {
     Call<Response<InlongStreamInfo>> getStream(@Query("groupId") String groupId,
             @Query("streamId") String streamId);
 
-    @POST("/stream/list")
+    @POST("stream/list")
     Call<Response<PageInfo<InlongStreamBriefInfo>>> listByCondition(@Body InlongStreamPageRequest request);
 
     @POST("stream/listAll")
     Call<Response<PageInfo<InlongStreamInfo>>> listStream(@Body InlongStreamPageRequest request);
 
-    @POST("/stream/startProcess/{groupId}/{streamId}")
```

Comment:
These have nothing to do with this ISSUE, mention ISSUE and PR separately

Reply:
OK , I have reset and then fix it in next PR.